debian/python3-pitop-full.install

```diff
@@ -0,0 +1,2 @@
+pitop/processing/algorithms/faces/predictors	/usr/lib/python3/dist-packages/pitop/processing/algorithms/faces/predictors
+pitop/processing/algorithms/faces/models      /usr/lib/python3/dist-packages/pitop/processing/algorithms/faces/models
```

Comment:
these should be removed once we tackle the large-files issue

Reply:
Do I just delete these lines or the whole file? Also, I've kept the onnx model in the repo but it is 2.6MB - maybe we should create a separate repo for this and download it like the dlib ones?

Reply:
Also, would you mind explaining what these lines are even doing?

Reply:
I've added the onnx model here: https://github.com/pi-top/pi-top-SDK-models